bincopy.py

```diff
@@ -611,9 +611,9 @@ def as_binary(self, minimum_address=None, padding=b'\xff'):
 
         for address, data in self.segments.iter():
             address //= self.word_size_bytes
-            res += padding * (address - current_maximum_address)
+            res += padding * (address - current_maximum_address)*self.word_size_bytes
```

Comment:
Giving the padding argument as `\xff\xff` should work instead of multiplying the padding by the word size. The question is, what does the user expect? In either case, the function doc-string should be changed to reflect the behaviour of the function.